cw/tvla.py

```diff
@@ -904,10 +904,11 @@ def run_tvla(ctx: typer.Context):
                     if cfg["mode"] == "aes":
                         # Simply plot everything.
                         samples = range(0, num_samples)
-                    else:
+                    elif cfg["mode"] == "kmac":
                         # For now, let's focus on the key absorption only.
                         samples = range(520, 2460)
-                        # Numbers for the eval branch:
+                    else:
+                        samples = range(100, 3200)
 
```

Comment:
I think we should add a comment to the file explaining the numbers.
It might seem clear now, but if later someone else has a look at these numbers a comment would help to understand the code faster. Especially for the else-case.

Reply:
This piece of code has a potential to get more confusing as we add more modes.
Perhaps it's time to simplify it with a dictionary indexed by `cfg["mode"]`?

Reply:
Good points, I'll add a comment and try to do a dictionary.

Reply:
Done.